sphinx/source/docs/user_guide/webgl.rst

```diff
@@ -27,10 +27,36 @@ Support
 -------
 
 Only a subset of Bokeh's objects are capable of rendering in WebGL. Currently
-supported are the circle and line glyphs, and many markers: asterisk, circle,
-square, diamond, triangle, inverted_triangle, cross, circle_cross, square_cross,
-diamond_cross, x, square_x, and circle_x. You can safely combine multiple glyphs
-in a plot, even if some are rendered in WebGL, and some are not.
+supported are the circle and line glyphs, and many markers:
+
+* asterisk
+
+* circle,
+
+* square
+
+* diamond
+
+* triangle
+
+* inverted_triangle
+
+* cross
+
+* circle_cross
+
+* square_cross
+
+* diamond_cross
+
+* x
+
+* square_x
+
+* circle_x
+
+You can safely combine multiple glyphs in a plot, even if some are rendered in WebGL,
+and some are not.
```

Comment:
This list is slightly out of date, the full current list of supported glyphs is the list below, plus "line":

https://github.com/bokeh/bokeh/blob/228132eba4b696b91b2a77f7e9d07771ba868093/bokehjs/src/lib/models/glyphs/webgl/markers.ts#L255-L268

so I think "hex" and "line" are missing. Other suggestions:

* the list should be alphabetized to make it a better reference 
* even better, the glyph names could link to the glyph methods in the reference guide

Reply:
@bryevdv thanks for the comment, that I can do! Just so I'm clear, when you say `link to the glyph methods in the reference guide`, do you mean these pages under `bokeh.models.markers`? https://docs.bokeh.org/en/latest/docs/reference/models/markers/asterisk.html#bokeh.models.markers.Asterisk

Reply:
sorry, confused `glyphs` and `markers`. I can link to both the glyphs and the markers.

Also, `line` is mentioned in the comment preceding the bulleted list: `Currently supported are the circle and line glyphs, and many markers:`

However, I see a `line` glyph, but no `circle` under the `bokeh.models.glyphs` reference. Is there an error there?

Reply:
@samwill Linking to the low-level glyph models is one option. Another option is to link the higher level glyph methods on `Figure`:

https://docs.bokeh.org/en/latest/docs/reference/plotting.html#bokeh.plotting.Figure.asterisk

I think i'd prefer the latter since the higher level interface is what most people should use most of the time, but either is acceptable and I leave the choice to you.  Either way, the links not use hard URLs in case we move things around later. The Sphinx syntax for that would normally be something like:
```
:func:`~bokeh.plotting.figure.Figure.asterisk`
```
But let me know if you have any questions!

Reply:
ah missed your comment and just pushed up a commit @bryevdv. I ended up linking to the low-level glyph models, but I did find a `Circle` glyph for the initial line that just needed a reference file created. However, I noticed that the circle glyph isn't mentioned under [models/glyphs/webgl](https://github.com/bokeh/bokeh/tree/228132eba4b696b91b2a77f7e9d07771ba868093/bokehjs/src/lib/models/glyphs/webgl). Should it be mentioned here at all, or is the correct list the `line` glyph and the `markers` mentioned?

Reply:
> However, I see a line glyph, but no circle under the bokeh.models.glyphs reference. Is there an error there?

@samwill not exactly wrong, but it's a little confusing for sure. Markers are a subset of glyphs, they all have the same properties: `x` and `y` coordinates and a `size` in pixels. They are all basically anything that you might use for a "scatter plot". Circles have all these same properties, and are very commonly used in scatter plots, but they also have a `radius` that can be specified in data-units. So they are are a bit in between the more specific "marker" category, and the more general "glyph" category. In terms of linking to it, the actual `Circle` definition is located in `markers.py` for uninteresting historical reasons.

Reply:
@samwill good question (and good eye for detail). `Circle` is webgl-enabled but it is not in that list at the bottom because it has to handled specially (specifically to account for the extra `radius` property that other markers do not have, e.g 

https://github.com/bokeh/bokeh/blob/228132eba4b696b91b2a77f7e9d07771ba868093/bokehjs/src/lib/models/glyphs/webgl/markers.ts#L230-L233

In any case, yes circle should included in the webgl page

Reply:
@bryevdv I see! Okay so current state is as follows: 

https://github.com/bokeh/bokeh/pull/10625/files#diff-6e03b8f3b32a5d5128a8505e2f954b20873b8312e08c038a1a9c50fdbd2737d3R30-R34

there's a mention of circle on line 30, and it's also included in the `markers` list on line 34. Should both mentions be there, or which should go?

Reply:
I left a comment inline, thanks for all the work on this!